cli/command/plugin/install.go

```diff
@@ -142,14 +130,14 @@ func runInstall(dockerCli *command.DockerCli, opts pluginOptions) error {
 		remote = trusted.String()
```

Comment:
I did this on purpose since it does not represent a tag but probably better to consistently use the familiar form when passing strings to the client. The daemon will always normalize values from the API anyway.